requirements-dev.txt

```diff
@@ -5,3 +5,7 @@ pre-commit
 pytest
 pytest-cov
 pytest-lazy-fixture
+datasets>=2.7.0
+torchvision>=0.12.0
+black
+build
```

Comment:
just curious how this is used?

Reply:
datasets and torchcvision are useful for writing code, since they are optional dependencies I have added them as dev requirements.
black for formatting - it is optional, since there's a hook for it too
build for building the package in a wheel